src/System.Private.Xml/src/System/Xml/Serialization/XmlSerializationWriter.cs

```diff
@@ -1853,7 +1854,7 @@ private string GenerateMembersElement(XmlMembersMapping xmlMembersMapping)
                         }
 
                         Writer.Write("if (pLength > ");
-                        Writer.Write(i.ToString(CultureInfo.InvariantCulture));
```

Comment:
Folks more knowledgeable about XML serialization can comment on this particular usage and whether there should be a follow-up change related to it.  For this PR, though, how the string is generated isn't changing, it's just being generated once instead of twice.